docs/user/aws/install_existing_vpc_local-zones.md

```diff
@@ -0,0 +1,427 @@
+- Check the total of subnets. If you choose 3 AZs to be created on the VPC stack, you should have 7 subnets on this list:
+
+```bash
+$ echo ${#SUBNETS[*]}
+7
+```
+
+##### Creating the subnet using AWS CLI <a name="create-network-subnet-res-cli"></a>
+
+TBD/Is it needed?
+
+##### Creating the subnet using AWS Console <a name="create-network-subnet-res-console"></a>
+
+TBD/Is it needed?
+
+## Install the cluster <a name="install-cluster"></a>
+
+To install the cluster in existing VPC with subnets in Local Zones, you should:
+- generate the install-config.yaml, you provide yours
+- add the subnet IDs setting the option `platform.aws.subnets`
+- (option) customize the Edge Machine Pool
+
+### Create the install-config.yaml <a name="create-config"></a>
+
+Create the `install-config.yaml` providing the subnet IDs recently created:
+
+- create the `install-config`
+
+```bash
+$ ./openshift-install create install-config --dir ${CLUSTER_NAME}
+? SSH Public Key /home/user/.ssh/id_rsa.pub
+? Platform aws
+? Region us-west-2
+? Base Domain devcluster.openshift.com
+? Cluster Name ipi-localzone
+? Pull Secret [? for help] **
+INFO Install-Config created in: ipi-localzone     
+```
+
+- Append the subnets to the `platform.aws.subnets`:
+
+```bash
+$ echo "    subnets:"; for SB in ${SUBNETS[*]}; do echo "    - $SB"; done
+    subnets:
+    - subnet-0fc845d8e30fdb431
+    - subnet-0a2675b7cbac2e537
+    - subnet-01c0ac400e1920b47
+    - subnet-0fee60966b7a93da6
+    - subnet-002b48c0a91c8c641
+    - subnet-093f00deb44ce81f4
+    - subnet-0f85ae65796e8d107
+```
+
+### Setting up the Edge Machine Pool <a name="create-config-edge-pool"></a>
+
+Version 4.12 e earlier introduces a new Machine Pool designed for the edge locations.
+The Edge Machine Pool configuration is common between edge locations, but due to the limitation
+of resources (Instance Types and Sizes) of the Local Zone, the default instance type created
+may vary from the traditional worker pool.
+
+The default EBS for Local Zone locations is `gp2`, different than the default worker pool.
+
+The preferred list of instance type follows the same order of worker pools, depending
+on the availability of the location, one of those instances will be chosen*:
+> Note: This list can be updated over the time
+- `m6i.xlarge`
+- `m5.xlarge`
+- `c5d.2xlarge`
+
+The Edge Machine Pool will also create new labels for the Machines to help developers to
+deploy their applications onto those locations. The new labels introduced are:
+    - `node-role.kubernetes.io/edge=''`
+    - `zone_type=local-zone`
+    - `zone_group=<Local Zone Group>`
+
+Finally, the MachineSets created on the Edge Machine Pool have `NoSchedule` taint to avoid the
+normal workloads spread out on those machines, and only user workloads will be allowed to run
+when the tolerations are defined on the pod spec.
+
+By default, the Edge Machine Pools will be created only when AWS Local Zone subnet IDs are added
+to the list of `platform.aws.subnets`.
```

Comment:
```suggestion
Finally, the Machine Sets created by the `edge` compute pool have `NoSchedule` taint to avoid the
regular workloads spread out on those machines, and only user workloads will be allowed to run
when the tolerations are defined on the pod spec.

By default, the `edge` compute pool will be created only when AWS Local Zone subnet IDs are added
to the list of `platform.aws.subnets`.
```